Compiler/include/Luau/BytecodeBuilder.h

```diff
@@ -166,12 +169,14 @@ class BytecodeBuilder
     struct ConstantKey
     {
         Constant::Type type;
-        // Note: this stores value* from Constant; when type is Number_Double, this stores the same bits as double does but in uint64_t.
+        // Note: this stores value* from Constant; when type is Type_Number, this stores the same bits as double does but in uint64_t.
+        // for Type_Vector, x and y are stored in value and z is stored in extra.
         uint64_t value;
+        uint32_t extra = 0;
```

Comment:
This unnecessarily increases the size of ConstantKey to 24 bytes.

Reply:
Thanks, I'll fix it on Monday.